spec/mobility/backends/active_record/container_spec.rb

```diff
@@ -20,6 +20,19 @@
     include_accessor_examples 'ContainerPost'
     include_dup_examples 'ContainerPost'
     include_cache_key_examples 'ContainerPost'
+
+    it 'does not change translations on access' do
+      post = ContainerPost.new
+
+      expect { post.title }.not_to change { post.translations }.from({})
+    end
+
+    it 'does not mix up dirty tracking on access' do
+      post = ContainerPost.new
+
+      expect { post.title }.not_to change { post.changes }.from({})
+      expect(post.changed?).to be(false)
+    end
```

Comment:
is there a better place for this specs? should they be some shared example for every backend?

Reply:
I think this is ok. This is just for general dirty tracking, correct? Not the Mobility dirty plugin. In which case it's a test that applies to any set of plugins using the container backend.